apps/api/src/app/benchmark/benchmark.service.ts

```diff
@@ -245,6 +245,45 @@ export class BenchmarkService {
     };
   }
 
+  public async deleteBenchmark({
+    dataSource,
+    symbol
+  }: UniqueAsset): Promise<Partial<SymbolProfile>> {
+    const assetProfile = await this.prismaService.symbolProfile.findFirst({
+      where: {
+        dataSource,
+        symbol
+      }
+    });
+
+    if (!assetProfile) {
+      return;
+    }
+
+    let benchmarks =
+      ((await this.propertyService.getByKey(
+        PROPERTY_BENCHMARKS
+      )) as BenchmarkProperty[]) ?? [];
+
+    benchmarks = benchmarks.filter(
+      (item) => item.symbolProfileId !== assetProfile.id
```

Comment:
Can you change to keep the style consistent?

```
({symbolProfileId}) => { return symbolProfileId !== assetProfile.id; }
```

Reply:
Resolved in latest commit.